servers/rendering/shader_language.h

```diff
@@ -700,6 +701,10 @@ class ShaderLanguage {
 			String group;
 			String subgroup;
 
+			_FORCE_INLINE_ bool is_texture() const {
+				return order < 0;
+			}
```

Comment:
I'd add a comment to explain why this works somehow, as it's not obvious at a glance.

Reply:
Done